_source/logzio_collections/_log-sources/python.md

```diff
@@ -84,6 +122,8 @@ you need to set every argument that comes before it.
 | timeout <span class="default-param">`3`</span> | Time to wait between log draining attempts, in seconds. |
 | listener-url <span class="default-param">`https://listener.logz.io:8071`</span> | Listener URL and port. <br> {% include log-shipping/replace-vars.html listener=true %} |
 | debug-flag <span class="default-param">`False`</span> | Debug flag. To print debug messages to stdout, `True`. Otherwise, `False`. |
+|Backup logs <span class="default-param">`True`</span>| If set to False, disables the local backup of logs in case of failure. |
```

Comment:
Just for preserving the existing convention, write them as `backup-logs` and `network-timeout` like the params that above them?